packages/libp2p/test/configuration/protocol-prefix.node.ts

```diff
@@ -67,7 +67,8 @@ describe('Protocol prefix is configurable', () => {
       '/ipfs/id/1.0.0',
       '/ipfs/id/push/1.0.0',
       '/ipfs/ping/1.0.0',
-      '/libp2p/fetch/0.0.1'
+      '/libp2p/fetch/0.0.1',
+      '/libp2p/circuit/relay/0.2.0/stop'
```

Comment:
Why did the extra protocol appear?

Reply:
Because we have configured a circuitRelayServer in the [aegir config](https://github.com/libp2p/js-libp2p/pull/1778/files/d45851d01f85b80dea647e0aaefa8371b753ec43#diff-9a79967796b063d2526c5bfdcf1b871c76f5f932863e48da9b2f8ceb59dba26eR56)